packaging/linux/run_keybase

```diff
@@ -157,8 +157,14 @@ run_redirector() {
 
   # Only start the root redirector if it hasn't been explicitly disabled.
   if [ "$disable" != "true" ]; then
-    logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
-    nohup keybase-redirector /keybase >> "$logdir/keybase.redirector.log" 2>&1 &
+    log="${XDG_CACHE_HOME:-$HOME/.cache}/keybase/keybase.redirector.log"
+    # An older version of post_install.sh could have made a redirector log
+    # here that's owned by root.  If we can't write to it, then just nuke it
+    # and overwrite.
+    if [ -e "$log" -a ! -w "$log" ]; then
+      rm -f "$log" &> /dev/null
```

Comment:
What output would `rm -f` print in this case?

Reply:
Nothing I know of, was just being prudent.

Reply:
Slight preference for leaving this un-suppressed. That way if we ever do get unexpected output, we'll have a chance of seeing it.